openspp-project/openspp-registry
# created a new module named spp_registry

## **Why is this change needed?**

There is no SPP Registry Base module that we can customize the registry generally.

## **How was the change implemented?**

Added a module named `spp_registry` with the following dependency
- g2p_registry_base
- g2p_registry_group
- g2p_registry_individual
- g2p_registry_membership

Also, moved some javascript code from other modules to this module

## **New unit tests**

TestRegistrant::test_get_import_templates

## **Unit tests executed by the author**

spp_registry/tests/test_res_partner::TestRegistrant

## **How to test manually**

- Go to Apps Menu.
- Click `Update Apps List` button in top part of the page
- Remove the Apps filter and search `spp_registry`.
- Check if `OpenSPP Registry: Base` module is showing.
- Check if it is installable.

## **Related links**

-
